packages/terra-date-input/src/DateInput.jsx

```diff
@@ -279,6 +279,12 @@ class DateInput extends React.Component {
       return;
     }
 
+    // prevent + and - characters from being entered into number input on keyDown
+    if (event.keyCode === 189 || event.keyCode === 187) {
```

Comment:
nitpick - can we stay consistent with how we reference keycodes by using the already imported KeyCode library? this also helps us avoid looking up what each code corresponds to. we could also cleanup the import by only importing the codes we need

Reply:
These 2 keycode values are not there in the Keycode js file

Reply:
hmm is KEY_DASH not equivalent to minus? i'm also confused on plus. doing a table lookup tells me 187 is equivalent to equals. 

this logic could also be extracted into a named helper

Reply:
Added keycode in utils file. 9f1f78c